package.json

```diff
@@ -53,12 +54,20 @@
     "start-hoc": "cd packages/light-hoc && yarn start",
     "start-react": "cd packages/fether-react && yarn start",
     "start-ui": "cd packages/fether-ui && yarn start",
-    "test": "lerna run test --parallel"
+    "test": "CI=true lerna run test --parallel"
```

Comment:
Why is CI=true necessary?

Reply:
Otherwise when running `yarn test` locally on the workspace, the test script of `fether-react` asks for user input:
```
fether-react: $ react-app-rewired test --env=jsdom
fether-react: No tests found related to files changed since last commit.
fether-react: Press `a` to run all tests, or run Jest with `--watchAll`.
fether-react: Watch Usage
fether-react:  › Press a to run all tests.
fether-react:  › Press p to filter by a filename regex pattern.
fether-react:  › Press t to filter by a test name regex pattern.
fether-react:  › Press q to quit watch mode.
fether-react:  › Press Enter to trigger a test run.
```
But apparently it cannot receive user input, so the test stalls

Reply:
(this change is not related to the PR)

Reply:
Ah OK! Got it, yes correct. Maybe we should add CI=true only in fether-react?

Reply:
Hmm but then we can't run the tests interactively on fether-react? If we run `yarn test` on fether-react it receives user input fine

Reply:
True, didn't think about that.